deepchem/feat/tests/test_basic.py

```diff
@@ -45,6 +47,9 @@ def testRDKitDescriptors(self):
     Test simple descriptors.
     """
     descriptors = self.engine([self.mol])
+    print(descriptors)
```

Comment:
Perhaps remove these print statements to keep logs clean.

Reply:
sure -- p.s. this was a real bug that I uncovered because of this test.